orchagent/acltable.h

```diff
@@ -27,6 +29,78 @@ typedef enum
     ACL_STAGE_EGRESS
 } acl_stage_type_t;
 
+typedef enum
+{
+    ACL_TABLE_UNKNOWN,
+    ACL_TABLE_L3,
+    ACL_TABLE_L3V6,
+    ACL_TABLE_MIRROR,
+    ACL_TABLE_MIRRORV6,
+    ACL_TABLE_MIRROR_DSCP,
+    ACL_TABLE_PFCWD,
+    ACL_TABLE_CTRLPLANE,
+    ACL_TABLE_DTEL_FLOW_WATCHLIST,
+    ACL_TABLE_DTEL_DROP_WATCHLIST
+} acl_table_type_t;
+
 typedef map<string, acl_stage_type_t> acl_stage_type_lookup_t;
 
-#endif /* SWSS_ACLTABLE_H */
+class AclOrch;
+class AclRule;
+
+class AclTable {
+    sai_object_id_t m_oid;
+    AclOrch *m_pAclOrch;
+public:
+    string id;
+    string description;
+    acl_table_type_t type;
+    acl_stage_type_t stage;
+
+    // Map port oid to group member oid
+    std::map<sai_object_id_t, sai_object_id_t> ports;
+    // Map rule name to rule data
+    map<string, shared_ptr<AclRule>> rules;
+    // Set to store the ACL table port alias
+    set<string> portSet;
+    // Set to store the not cofigured ACL table port alias
+    set<string> pendingPortSet;
```

Comment:
Probably doesn't make a huge difference since we're not storing very much data, but can we use unordered maps/sets here instead of the normal maps/sets?